.github/workflows/appimage.yml

```diff
@@ -0,0 +1,53 @@
+name: AppImage
+
+on:
+  pull_request:
+    branches: [master, features]
+  release:
+
+jobs:
+  build-appimage:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Prepare environment
```

Comment:
Alternatively, you could use this existing Action: https://github.com/marketplace/actions/build-appimage

Reply:
We'll stick on the `make` version for now, and move to it as soon as [the Dockerfile](https://github.com/AppImageCrafters/appimage-builder/blob/master/Dockerfile) updated to a version of Ubuntu with a more recent version of Python.
Thanks for pointing it :+1: